tests/astrapy/test_db_dml.py

```diff
@@ -362,6 +365,50 @@ def test_insert_many_ordered_false(
     assert check_response["data"]["document"]["_id"] == _id1
 
 
+@pytest.mark.describe("test error handling")
+def test_error_handling(writable_vector_collection: AstraDBCollection) -> None:
+    _id1 = str(uuid.uuid4())
+
+    result1 = writable_vector_collection.insert_one(
```

Comment:
can you please also add a test about a network error ? like unreachable server (just configure a dummy url like localhost:1234)